numba/tests/test_lists.py

```diff
@@ -692,8 +693,10 @@ def test_mul_error(self):
         pyfunc = list_mul
         cfunc = jit(nopython=True)(pyfunc)
         # Fail in malloc()
-        with self.assertRaises(MemoryError):
+        with self.assertRaises(MemoryError) as cm:
             cfunc(1, 2**58)
+        libc = ct.CDLL('libc.dylib')
+        libc.printf("###Please ignore above error message i.e. can't allocate region. It is in fact purpose of the this test to overflow memory###\n".encode("UTF-8"))
```

Comment:
This line needs the suggested change below made, it also needs wrapping to 80 chars, thanks.
```suggestion
        libc.printf("###Please ignore above error message i.e. can't allocate region. It is in fact the purpose of this test to request more memory than can be provided###\n".encode("UTF-8"))
```